ckanext/switzerland/dcat/harvesters.py

```diff
@@ -58,6 +70,19 @@ def _get_guid(self, dataset_dict, source_url=None):  # noqa
                             'config (%s)' % (org.id, dataset_dict['owner_org'])
                         )
                         return None
+
+                    if self.config.get('excluded-dataset-identifiers'):
+                        dataset_identifier = guid.split('@')[0]
+                        excluded_dataset_identifiers = self.config.get('excluded-dataset-identifiers')  # noqa
+                        if dataset_identifier in excluded_dataset_identifiers:
+                            log.error(
+                                'The dataset with identifier (%s) is ignored '
+                                'as configured in the harvester-config in '
+                                '"excluded-dataset-identifiers"'
+                                % (dataset_dict['identifier'])
+                            )
+                            return None
```

Comment:
It's a simple and elegant solution to implement this here, as we're already parsing the datasets and we are in the gather stage. But be aware, if you simply return `None` here, [this code](https://github.com/ckan/ckanext-dcat/blob/a492acb156712d5cb644350ff8731f6809f76311/ckanext/dcat/harvesters/rdf.py#L220-L222) will kick in. So all he sees is in the Dashboard is `"Could not get a unique identifier for dataset: XY"`

I would either add my own gather_error to explain what's wrong. But this is actually not my favourite choice. Because nothing is wrong, everything works as expected, so an error should not be raised. So I would suggest to handle this whole code not in the _get_guid() method, where it is kind-of a hack, but do it properly in the `after_download` provided by the `IDCATRDFHarvester` interface. WDYT?